homeassistant/util/dt.py

```diff
@@ -109,6 +109,15 @@ def str_to_datetime(dt_str):
         return None
 
 
+def timestamp_to_short_time_str(timestamp):
```

Comment:
This looks like a very specialized use case that also is not timezone aware.

Can you use the `utc_from_timestamp` and `datetime_to_short_time_str` methods?